lib/xclarity_client/config_pattern.rb

```diff
@@ -0,0 +1,18 @@
+module XClarityClient
+  class ConfigPattern
+    include XClarityClient::Resource
+
+    BASE_URI = '/patterns'.freeze
+    LIST_NAME = 'items'.freeze
+
+    attr_accessor :id, :referencedBy, :userDefined, :formFactor, :inUse, :description, :name, :nodeType, :type,
```

Comment:
I notice that _ConfigProfile_ has _identifier_, _items_, and _label_ attributes. Are they also needed here?

Reply:
included